toolkit/governed-map/primitives/src/lib.rs

```diff
@@ -284,6 +279,13 @@ impl sp_inherents::InherentDataProvider for GovernedMapInherentDataProvider {
 #[cfg(feature = "std")]
 #[async_trait::async_trait]
 pub trait GovernedMapDataSource {
+	/// Returns all key-value mappings stored in the Governed Map on Cardano after execution of `mc_block`.
+	async fn get_current_mappings(
```

Comment:
```suggestion
	async fn get_state_at_block(
```
state vs mappings => I would like to have one name for this across all layers. Runtime api has "get_current_state".

"current" vs "at_block" => because current could mean at the time of invoking this function